### PR TITLE
Fixed devise oauth image issue with facebook

### PR DESCRIPTION
**Pull request by otownsend92**

### CODE REVIEW

**Comment by otownsend92** [comment]
+1

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -236,7 +236,7 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-  config.omniauth :facebook, "1495604120735583", "4adef8998ba66e2643a8258f2c74a312", scope: 'email', info_fields: 'email, name, image'
+  config.omniauth :facebook, "1495604120735583", "4adef8998ba66e2643a8258f2c74a312", scope: 'email', info_fields: 'email, name'
```
Comment (bboe) [inline config/initializers/devise.rb:239]: Is it okay that this APP_SECRET is publicly available?

Comment (otownsend92) [inline config/initializers/devise.rb:239]: Good catch, thank you. Do you happen to know how we might hide this from GitHub yet somehow include it in the master branch for deploying on production?

Comment (bboe) [inline config/initializers/devise.rb:239]: Pushing keys to servers can be a pain in the ass. You have a few options from simple, but tedious, to more complex, but automated:

(1) Manually update your application server code with each deploy, and relaunch the application server process
(2) Automate the above process with a script that you need only provide a list of IP addresses for
(3) Customize the cloudformation templates inject the keys into the file prior to launching the server

The third option is the most robust, but you'd have to alter each of the different cloudformation templates. Fortunately it should be the same change to each.

**Pull request merged**